I pushed a change to `dev/test.sh` to silence warnings and errors from [shellcheck](https://www.shellcheck.net/).

The errors it flagged caused no problems in practice. The most helpful error msg was this one, on why not to mix eval with arrays:
https://www.shellcheck.net/wiki/SC2294

The other catch was that nested quotes actually led to the `$i` argument to be unquoted:
https://github.com/saulpw/visidata/blob/aad595dc5dceaabfd9ec716428c6c00ffe91e625/dev/test.sh#L59

--- a/.github/workflows/main.yml
+++ b/.github/workflows/main.yml
@@ -40,12 +40,14 @@ jobs:
     - name: Ensure VisiData can load a dir
       run: vd . --batch
 
-    - name: Install lxml dependencies
-      run: sudo apt install -y libxml2-dev libxslt-dev
-
-    - name: Install ping dependencies
-      run: sudo apt install -y traceroute
-
+# Apt packages are slow to install, often taking up to a minute.
+# lxml dependencies are unneeded when installing lxml via pip
+#    - name: Install lxml dependencies
+#      run: sudo apt install -y libxml2-dev libxslt-dev
+# only needed for testing open-ping, which isn't tested currently anyway
+#    - name: Install ping dependencies
+#      run: sudo apt install -y traceroute
+#
     - name: Install optional dependencies
       run: |
         pip3 install ".[test]"

